pandas/tests/extension/base/interface.py

```diff
@@ -29,6 +29,31 @@ def test_can_hold_na_valid(self, data):
         # GH-20761
         assert data._can_hold_na is True
 
+    def test_contains(self, data, data_missing):
+        # GH-37867
+        # Tests for membership checks. Membership checks for nan-likes is tricky and
+        # the settled on rule is: `nan_like in arr` is True if nan_like is
+        # arr.dtype.na_value and arr.isna().any() is True. Else the check returns False.
+
+        for this_data in [data, data_missing]:
```

Comment:
see my comments below on how to actually parameterize this

Reply:
I don't its possible to parametrize fixtures? do have an example or hint on how that's done?

Reply:
grep for

pytest.getfixturevalue in tests
we just had a use of this

Reply:
Couldn't get it to work, I'll try it again tonight.

Reply:
Sorry, but I would simply leave it as is, using `pytest.getfixturevalue ` only makes it way more complicated as it needs to be